README.md

```diff
@@ -78,7 +75,7 @@ The output hdf5 files each contain two datasets:
 | `merges_file` | str | None | Valid file path | The merges file to be used with the tokenizer class specified by `tokenizer_class`. If `pretrained_tokenizer` tokenizer is not specified, this is required. It should be a .txt file for a GPT2 tokenizer. |
 | `special_tokens_dict` | str | None | string representation of json | Any non-standard special tokens in JSON format to add to tokenizer. e.g. \"{'sep_token': \"[SEP]\"}\". |
 | `max_seq_length` | int | 2048 | 512 for gpt2 small, 1024 for gpt-xl, 2048 for gpt3-13B. | The maximum sequence length of the model you are using. |
-| `input_packing_style` | str | 'full' | ['full', 'single_truncate_overflow', 'single_drop_overflow', 'greedy'] | 'full': Completely fill sequences with tokens, as soon as sequences is full start packing into new sequence. Ignore article boundaries, they may be split across multiple sequences. 'greedy' Fit as many articles as possible into a sequence, make sure no article is split across multiple sequences. Fill the left over space in each sequence with padding. If an article is longer than the maximum sequence length, throw it out. 'single_drop_overflow' Each sequence contains only 1 article.  Fill the rest of the sequence with padding.  If an article is longer than the maximum sequence length, throw out the article. 'single_truncate_overflow': Each sequence contains only 1 article.  Fill the rest of the sequence with padding.  If an article is longer than the maximum sequence length truncate it, and throw the remaining portion of the article out. |
+| `input_packing_config` | PackingConfig | 'full' | ['full', 'single::truncate_left', 'single::truncate_right', 'single::drop', 'greedy::truncate_left', 'greedy::truncate_right', 'greedy::drop'] | 'full': Completely fill sequences with tokens, as soon as sequences is full start packing into new sequence. Ignore article boundaries, they may be split across multiple sequences. 'greedy': Fit as many articles as possible into a sequence, make sure no article is split across multiple sequences. Fill the left over space in each sequence with padding.  'single': Each sequence contains only 1 article.  Fill the rest of the sequence with padding.  'drop': Drop the entire article if there are any tokens that overflow beyond the max sequence length.  'truncate_left':  Truncate the article from the left if there are any tokens that overflow beyond the max sequence length.  'truncate_right':  Truncate the article from the right if there are any tokens that overflow beyond the max sequence length. |
```

Comment:
Maybe we should add in more detail on what the part of the argument before :: means and after. 

Also for the future I think we should add an entire section in the readme on input_packing_config - it is the most important and complicated flag